aiida/engine/processes/process.py

```diff
@@ -121,6 +121,22 @@ def exit_codes(self):
         """
         return self.spec().exit_codes
 
+    @classproperty
+    def spec_metadata(self):
+        """Return the metadata port namespace of the process specification of this process.
+
+        :return: metadata dictionary
+        """
+        return self.spec().inputs['metadata']
+
+    @classproperty
+    def spec_options(self):
```

Comment:
perhaps replace with cls for clarity?

Reply:
I did, but then pylint complains because it does not understand our `classproperty` decorator. I think across the code base we now use `self` instead of `cls`. Maybe it is indeed better to just add the `pylint` disable statement

Reply:
I have changed `self` to `cls`